src/app/drive/components/DeleteBackupDialog/DeleteBackupDialog.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import errorService from 'app/core/services/error.service';
+import { uiActions } from 'app/store/slices/ui';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import { RootState } from 'app/store';
+import { DriveFolderData as DriveWebFolderData, DriveItemData } from '../../types';
+import i18n from 'app/i18n/services/i18n.service';
+import Button from 'app/shared/components/Button/Button';
+import Modal from 'app/shared/components/Modal';
+import { deleteItemsThunk } from '../../../store/slices/storage/storage.thunks/deleteItemsThunk';
+import { deleteBackupDeviceAsFolder } from '../../../drive/services/folder.service';
+import { backupsThunks } from 'app/store/slices/backups';
+import { SdkFactory } from '../../../core/factory/sdk';
+
+export interface BreadcrumbItemData {
+  id: number;
+  label: string;
+  icon: JSX.Element | null;
+  active: boolean;
+  isFirstPath?: boolean;
+  dialog?: boolean;
+  isBackup?: boolean;
+  onClick?: () => void;
+}
+
+interface BreadcrumbsProps {
```

Comment:
Wrong name, in any case `DeleteBackupDialogProps`

Reply:
fixed